src/mlpack/bindings/cli/third_party/CLI/CLI11.hpp

```diff
@@ -691,7 +691,7 @@ class CallForVersion : public Success {
         : CallForVersion("This should be caught in your main function, see examples", ExitCodes::Success) {}
 };
 
-/// Does not output a diagnostic in CLI11_PARSE, but allows to return from main() with a specific error code.
+/// Does not output a diagnostic in CLI11_PARSE, but allows main() to return with a specific error code.
```

Comment:
I guess since this is a third party header fill, we should fix this upstream.

Reply:
https://github.com/CLIUtils/CLI11/pull/510